### PR TITLE
update lindberg2022simple_ontorights

### PR DESCRIPTION
**Pull request by pedropaulofb**
OntoRights' Simple (now 'Easy') version was updated from version 1.0 to 1.1.

### CODE REVIEW

File: models/lindeberg2022simple-ontorights/metadata.yaml [hunk 1]
```diff
@@ -17,6 +18,8 @@ context:
  - research 
 source:
  - https://github.com/JoranL/human-rights-ontology/blob/gh-pages/files/Lindeberg%202022%20Ontology%20Design%20Master%20Thesis.pdf
```
Comment (tgoprince) [inline models/lindeberg2022simple-ontorights/metadata.yaml:20]: This link is broken.

File: models/lindeberg2022simple-ontorights/metadata.yaml [hunk 2]
```diff
@@ -1,13 +1,14 @@
-title: Simple OntoRights
+title: Easy OntoRights
 acronym:
 issued: 2022
-modified:
+modified: 2023
```
Comment (tgoprince) [inline models/lindeberg2022simple-ontorights/metadata.yaml:4]: Shouldn't this be 2024? Given that the latest paper has not even been published yet?

**Review by tgoprince — CHANGES_REQUESTED** [review]

File: models/lindeberg2022simple-ontorights/metadata.yaml (hunk 2 above)
Comment (pedropaulofb) [reply models/lindeberg2022simple-ontorights/metadata.yaml:4]: I used the publication date provided on the [ontology's landing page](https://joranl.github.io/human-rights-ontology/ontorights-access.html), as I believe it is the most reliable date, reflecting when it was truly made available to users.

**Review by tgoprince — APPROVED** [review]

**Comment by tgoprince** [comment]
Please resolve the conflicts so we can merge this PR.

**Comment by pedropaulofb** [comment]
Fixes done! PR Squashed and merged.

> Please resolve the conflicts so we can merge this PR.

**Pull request merged**